### PR TITLE
Publish

### PR DESCRIPTION
**Pull request by pokornyd**
### Motivation

Fixes typos and orphaned comments, adds github files to comply with community standards.

### Checklist

- [ ] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [ ] Tests are passing
- [ ] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by Simply007** [comment]
We sould validate text in code of conduct and contributing (and possibly create a repo template for theese) to include License and Readme structure. Not sure if part of the PR, I might rather split it into the task.

**Comment by pokornyd** [comment]
> 

I used the templates from previous org but it will require some updates. created a task for new repo template

**Comment by Simply007** [comment]
So are we going to adjust the files now for this one, or remove them from this PR?

**Comment by pokornyd** [comment]
> So are we going to adjust the files now for this one, or remove them from this PR?

I added the [updated repo template](https://github.com/kontent-ai/repo-template) we can use, though there are still some matters left to discuss (devrel contact, stackoverflow tags..)

**Comment by pokornyd** [comment]
removed contributing and code of conduct from the PR

File: package.json [hunk 1]
```diff
@@ -56,6 +55,9 @@
     "typescript": "^4.5.5",
     "webpack": "^5.68.0"
   },
+  "peerDependencies": {
+    "@kentico/kontent-delivery": "11.x"
```
Comment (Simply007) [inline package.json:60]: Just add `@kentico/kontent-delivery` latest with ^´notation to dev dependencies

**Review by Simply007 — CHANGES_REQUESTED** [review]
Just one bit and it is ready to be merged.

File: package.json (hunk 1 above)
Comment (pokornyd) [reply package.json:60]: added

**Review by Simply007 — APPROVED** [review]

**Pull request merged**